From: Tim Weippert <weiti@weiti.org>
Date: Sun, 13 Feb 2011 14:14:27 +0100
Subject: [PATCH] man page fixes

Fix some spelling errors and hyphen vs minus errors in manpages
---
 docs/man/c-icap-mkbdb.8.in |    4 ++--
 docs/man/c-icap.8.in       |    6 +++---
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/docs/man/c-icap-mkbdb.8.in b/docs/man/c-icap-mkbdb.8.in
index 103bae4..c50bc1a 100644
--- a/docs/man/c-icap-mkbdb.8.in
+++ b/docs/man/c-icap-mkbdb.8.in
@@ -47,12 +47,12 @@ The type of the values. The "string" is the default.
 Do not update the database just dump it to the screen.
 .SH EXAMPLES
 .TP 
-c-icap-mkbdb -o keys -i keys.txt
+c-icap-mkbdb \-o keys \-i keys.txt
 It builds the
 .B keys 
 Berkeley DB database, using string type for keys and values.
 .TP
-c-icap-mkbdb -o keys --dump
+c-icap-mkbdb \-o keys --dump
 Dump the contents of the keys database
 .SH SEE ALSO
 .BR c-icap "(8)"
diff --git a/docs/man/c-icap.8.in b/docs/man/c-icap.8.in
index 79585f0..f7f639b 100644
--- a/docs/man/c-icap.8.in
+++ b/docs/man/c-icap.8.in
@@ -42,7 +42,7 @@ By default c-icap writes its pid in this file. The path of this file can changed
 .RE
 .I /var/run/c-icap.ctl
 .RS
-The commands socket. This file used to send commands to the icap server from command line. For informations about implemented commands look below in the "Implemented commands" sub-section
+The commands socket. This file used to send commands to the icap server from command line. For information about implemented commands look below in the "Implemented commands" sub-section
 .SH NOTES
 .SS Implemented commands
 Currently the following commands are implemented:
@@ -60,7 +60,7 @@ Services and modules can define their own commands.
 .PP
 \fBExamples:\fR
 .IP "To reconfigure c-icap:"
-      echo -n "reconfigure" > /var/run/c-icap.ctl
+      echo \-n "reconfigure" > /var/run/c-icap.ctl
 .RE
 .SS Lookup tables
 Lookup tables are simple read-only databases. 
@@ -106,7 +106,7 @@ Someone can retrieve runtime information using the \fBinfo\fR service. The infor
 .PP
 \fBExample:\fR
 .IP "Retrieve runtime information from command line:"
-	c-icap-client -i localhost -s "info?view=text" -req "a_url"
+	c-icap-client \-i localhost \-s "info?view=text" \-req "a_url"
 .RE
 .SH SEE ALSO
 .BR c-icap-client "(8)"
-- 
